docs/release_docs/AHA_Platform_Deployment_Workflows.md

```diff
@@ -0,0 +1,74 @@
+# AHA Platform Deployment Workflows
+
+## AHA-PROD-CI-CD
+Mode of trigger: ```On-Demand```
```

Comment:
In the case of the on-demand trigger, what workflow parameters must the user provide to run the workflow?
Can we add these details to all such On-Demand workflows